### PR TITLE
front: rollingstockeditor: add basePowerClass in payload for post and patch

### LINKED ISSUE #5119: Duplicated rolling stock won't use electrical profiles
### What happened?

I duplicated a rolling stock and tried to create two trains : one with the original one and one with the duplicate.
It appears that the duplicate isn't using any electrical profiles.
https://github.com/osrd-project/osrd/assets/86296016/598b2463-9226-4e30-9720-371d0ddcf4b6

### What did you expect to happen?

The duplicated rolling stock should behave the exact same way as the original one.

### How can we reproduce it (as minimally and precisely as possible)?

1. Go to the scenario that's in the video

### What operating system, browser and environment are you using?

- **Browser**: Chrome
- **OS**: Windows 11
- **Env**: dev

### OSRD version (top right corner `Account` button > `Informations`)

ee04bf08

### PR DESCRIPTION
**Pull request by Yohh**
close #5119

### CODE REVIEW

File: front/src/modules/rollingStock/helpers/utils.ts [hunk 1]
```diff
@@ -159,6 +160,7 @@ export const rollingStockEditorQueryArg = (
     unit: data.unit,
   },
   effort_curves: currentRsEffortCurve,
+  base_power_class: (data.basePowerClass as string) || '',
```
Comment (clarani) [inline front/src/modules/rollingStock/helpers/utils.ts]: No need for the cast here
```suggestion
  base_power_class: data.basePowerClass || '',
```

**Review by clarani — CHANGES_REQUESTED** [review]
Nice thanks, only 1 comment !

File: front/src/modules/rollingStock/helpers/utils.ts (hunk 1 above)
Comment (Yohh) [reply front/src/modules/rollingStock/helpers/utils.ts]: done

**Review by SharglutDev — APPROVED** [review]
Tested in local with duplicated rs. Good job

**Review by clarani — APPROVED** [review]
LGTM and tested !

**Pull request merged**